packages/react/src/components/Table/Table.test.jsx

```diff
@@ -1369,7 +1298,7 @@ describe('Table', () => {
       />
     );
 
-    expect(screen.queryAllByTestId('table-head--overflow').length).toBe(5);
+    expect(screen.queryAllByTestId('table-head--overflow').length).toBe(8);
```

Comment:
Should we have any changes to existing unit test like this?

Reply:
In general no, but this stems from that I wanted to share the test data between the unit and the e2e tests and for the e2e I needed 3 additional columns so I added those to the `getColumns` function in the Table.test.helpers.jsx. Hence we went from 5 to 8 columns in our tests and I had to adjust this one unit test.